assignment2.yaml

```diff
@@ -0,0 +1,138 @@
+swagger: "2.0"
+info:
+  description: "Welcome to the swapi, the Star Wars API! This documentation should help you familiarise yourself with the resources available and how to consume them with HTTP requests. If you're after a native helper library then I suggest you scroll down and check out what's available. Read through the getting started section before you dive in. Most of your problems should be solved just by reading through it."
+  version: ""
+  title: "SWAPI"
+host: "swapi.co"
+basePath: "/documentation"
+tags:
+- name: "Films"
+  description: "A Film resource is a single film."
+schemes:
+- "http"
+paths:
+  /films/:
+    get:
+      tags:
+      - "Films"
+      summary: "get all the film resources"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - name: ":id"
+        in: "query"
+        description: "id of the film to be returned"
+        required: true
+        type: "integer"
+      responses:
+        200:
+          description: "OK"
+          schema:
+            $ref: "#/definitions/Films"
+  /films/{:id}/:
+    get:
+      tags:
+      - "Films"
+      summary: "get a specific film resource"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - name: ":id"
+        in: "path"
```

Comment:
how can you pass filmId as a parameter in a _film listing_ API! It is supposed to return you with all the film resources, not a specific resource with a particular Id